src/strings/en-gb.json

```diff
@@ -1007,8 +1007,9 @@
     "LabelArtistsHelp": "Separate multiple artists with a semicolon.",
     "LabelArtists": "Artists",
     "LabelAppName": "App name",
-    "LabelAllowedRemoteAddressesMode": "Remote IP address filter mode",
-    "LabelAllowedRemoteAddresses": "Remote IP address filter",
+    "LabelAllowContentWithTags": "Allow items with tags:",
+    "LabelAllowedRemoteAddressesMode": "Remote IP address filter mode:",
+    "LabelAllowedRemoteAddresses": "Remote IP address filter:",
```

Comment:
Only the `en-us` language should be updated in PRs otherwise it can cause conflicts with weblate (translate.jellyfin.org).